packages/circularprogress/src/react/index.tsx

```diff
@@ -80,18 +81,44 @@ const CircularProgress = React.forwardRef<
   CircularProgressProps
 >((props, ref) => {
   const defaultIndeterminateValue = 25
-  const { size, value = defaultIndeterminateValue, ...rest } = props
+  const { size, value, 'aria-label': ariaLabel, ...rest } = props
   const themeName = useTheme()
 
-  const dashOffset = ((100 - value) / 100) * circumference
-
+  const dashOffset =
+    ((100 - (value || defaultIndeterminateValue)) / 100) * circumference
+  const [busy, setBusy] = React.useState<'false' | 'true'>('false')
+  React.useEffect(() => {
+    if (value) {
+      value > 0 && setBusy('true')
+      value === 100 && setBusy('false')
+    }
+    setBusy('true')
+  }, [value])
```

Comment:
```suggestion
  const busy = value 
    ? value === 100
      ? 'false'
      : value > 0
        ? 'true'
        : 'false'
    : 'true'
```

Don't need state or effect. Could have same functionality with value derived from props.